sdcm/cluster.py

```diff
@@ -5809,6 +5809,7 @@ def start_scylla_monitoring(self, node):
         self.add_sct_dashboards_to_grafana(node)
         self.save_sct_dashboards_config(node)
         self.save_monitoring_version(node)
+        remove_if_exists(self.sct_dashboard_json_file)
```

Comment:
I would recommend using: 
https://docs.python.org/3/library/pathlib.html#pathlib.Path.unlink

and remove completely `remove_if_exists`

Reply:
in general we should shy away from `os.path`, at some day it would be deprecated